e2e/test/scenarios/metrics/browse.cy.spec.ts

```diff
@@ -230,4 +231,60 @@ describe("scenarios > browse > metrics", () => {
       getMetricsTableItem(3).should("contain", "Metric D");
     });
   });
+
+  describe("dot menu", () => {
+    it("should be possible to bookmark a metrics from the dot menu", () => {
+      createMetrics([ORDERS_SCALAR_METRIC]);
+
+      cy.visit("/browse/metrics");
+
+      metricsTable().findByLabelText("Metric options").click();
+      popover().findByText("Bookmark").should("be.visible").click();
+
+      metricsTable().findByLabelText("Metric options").click();
+      popover()
+        .findByText("Remove from bookmarks")
+        .should("be.visible")
+        .click();
+
+      metricsTable().findByLabelText("Metric options").click();
+      popover().findByText("Bookmark").should("be.visible");
+    });
+
+    it("should be possible to navigate to the collection from the dot menu", () => {
+      createMetrics([ORDERS_SCALAR_METRIC]);
+
+      cy.visit("/browse/metrics");
+
+      metricsTable().findByLabelText("Metric options").click();
+      popover().findByText("Open collection").should("be.visible").click();
+
+      cy.location("pathname").should("eq", "/collection/root");
+    });
+
+    it("should be possible to trash a metric from the dot menu when the user has write access", () => {
```

Comment:
Please check that the metric is available in trash and can be restored; it should appear back here without refreshing the page etc. Also check undo.